### PR TITLE
[PD-3683] Hide "Goal Minus -" from Users

### PR DESCRIPTION
**Pull request by vigowei**
## What's this PR do?
Instead of displaying Goal Minus One/Two, we are providing more context to those labels.

## Screenshots (if appropriate)
image

image

## Where should the reviewer start?
myreports/datasources/util.py L#424

## How should this be manually tested?
Go to "Goal Flow" in site traffic v2 report, drilldown and notice the first column is labelled correctly, you should no longer see "Goal Minus *" anywhere, including exported csv/excel

### Front-End

- [x] Works on IE 11 and Edge
- [x] Works on Mobile

### Separation of Concerns

- [x] Back end "business logic" is implemented in plain python, Django model classes, and Django forms. (not Django views or other frameworks)
- [x] Concerns of UI, api, and model are in separate code units.

### CODE REVIEW

File: myreports/presentation/xlsx.py [hunk 1]
```diff
@@ -161,14 +167,18 @@ def write_presentation(self, values, records, output, report=None,
...
+
+        for column, value in enumerate(header_values):
             worksheet.write_string(0, column, value)
 
         for row, record in enumerate(records):
             value_iterator = enumerate([unicode(record.get(v)) for v in values])
             for column, text in value_iterator:
                 worksheet.write_string(row + 1, column, text)
-        # write timeline data into a sheet of workbook
+        # writeq timeline data into a sheet of workbook
```
Comment (tdphillips) [inline myreports/presentation/xlsx.py]: Typo. I know it's just a comment.

Comment (vigowei) [reply myreports/presentation/xlsx.py]: Thanks for the catch. Fixed!

**Comment by aljordan82** [comment]
I am getting an error when xporting goal-flow
`KeyError at /analytics/api/export/`

**Comment by aljordan82** [comment]
Export now works, but the Sheet title is just "Data"

**Comment by aljordan82** [comment]
I've test this PR and it works.

**Pull request merged**